icu4c/source/test/intltest/measfmttest.cpp

```diff
@@ -3549,6 +3551,29 @@ void MeasureFormatTest::Test21060_AddressSanitizerProblem() {
     status.errIfFailureAndReset();
 }
 
+void MeasureFormatTest::Test21223_FrenchDuration() {
+    IcuTestErrorCode status(*this, "Test21223_FrenchDuration");
+    MeasureFormat mf("fr-FR", UMEASFMT_WIDTH_NARROW, status);
+    Measure H5M10[] = {
+        {5, MeasureUnit::createHour(status), status},
+        {10, MeasureUnit::createMinute(status), status}
+    };
+    UnicodeString result;
+    FieldPosition pos;
+    mf.formatMeasures(H5M10, UPRV_LENGTHOF(H5M10), result, pos, status);
+    assertEquals("Should have consistent spacing", u"5h 10min", result);
+
+    // Test additional locales:
```

Comment:
Is this a placeholder for future tests for other locales that use narrow-nbsp in their duration formatting?

Reply:
It's intended as a way to spot-check for other languages that might have a similar problem as French.  Uncomment the lines and look at the output.  I could delete it, but I see no reason to.